### DISCUSSION TITLE
Help! Does anyone have the v 2.4.6 for Mac? I could really use it. Thanks.

### DISCUSSION THREAD

**Discussion by Frederiknis** [artisan-roaster-scope/artisan#1742] [Q&A]
Please help. 
V. 2.4.6 for Mac

**Comment by roasterdave** [comment]
Why?  That version is several years old, not available for download and not supported.  If there is a problem with the current version please let us know so we can work to resolve it.

**Comment by Frederiknis** [comment]
I’m currently operating with the Kaleido M2 and have come across issues with the connection to the cooler fan, which I didn’t have with 2.4.6. (Previous version  that I used) I’ve tried to rewrite the syntax in the description under Config->Events->Buttons but the connection to my cooler fan is dead. Connection with Bluetooth is no problem and access to heater, drum and air is not a problemMvh Frederik Nis-HanssenDen 16. nov. 2024 kl. 17.38 skrev Dave Baxter ***@***.***>:﻿
Why?  That version is several years old, not available for download and not supported.  If there is a problem with the current version please let us know so we can work to resolve it.

**Comment by Terracotta-6** [comment]
Cooling on Serial Command  CLDN,ON  
Cooling off Serial Command  CLDN,OFF

**Comment by Frederiknis** [comment]
Negative. Made it work but with OT4,{} Any place one can see all commands available for Artisan with Kaleido M2? Mvh Frederik Nis-HanssenDen 17. nov. 2024 kl. 10.10 skrev Terracotta-6 ***@***.***>:﻿
Cooling on Serial Command  CLDN,ON
Cooling off Serial Command  CLDN,OFF

**Reply by Terracotta-6** [reply]
Port OT4 Kaleido is not enabled
